src/api/webhooks.js

```diff
@@ -27,8 +27,14 @@ export class WebhooksApi extends BaseApi {
             return Promise.reject(new Error('Must provide props.'));
         }
 
-        if (isTargetRequired && !props.target) {
-            return Promise.reject(new Error('Must provide a target.'));
+        if (isTargetRequired) {
+            if (!props.target) {
+                return Promise.reject(new Error('Must provide a target.'));
+            }
+
+            if (!props.target.startsWith('http://') || !props.target.startsWith('https://')) {
```

Comment:
Fixed and passing on [integration tests](https://circleci.com/gh/smooch/smooch-core-js/183?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).